dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubHelper.java

```diff
@@ -42,10 +46,58 @@
 public final class GitHubHelper {
 
     private static final Logger LOG = LoggerFactory.getLogger(GitHubHelper.class);
+    public static final String BRANCHNAME_FOR_BOT = "feature/add_dockstore_yml";
+
+    public static final String DOCKSTORE_BOT_PR_TEXT = """
+           The dockstore-bot has guessed at a .dockstore.yml for your repository.
+           
+           Please review, make appropriate changes, add any [additional keys](https://docs.dockstore.org/en/stable/assets/templates/template.html) and merge in order to complete integration with Dockstore, allowing Dockstore to keep informed of new changes to your workflow(s), notebook(s), or tool(s).
+           """;
+
+    public static final String DOCKSTORE_BOT_PR_TITLE =  "dockstore-bot inferred .dockstore.yml";
```

Comment:
Suggesting that the PR title start with a verb like "Add dockstore-bot inferred .dockstore.yml". Could even shorten it to "Add .dockstore.yml" because the PR description will say that the dockstore-bot guessed a .dockstore.yml